ocs_ci/ocs/constants.py

```diff
@@ -759,6 +759,7 @@
     '{ "op": "add", "path": "/data/CSI_ENABLE_VOLUME_REPLICATION", "value": "true" }]\''
 )
 RBD_SIDECAR_COUNT = 18
+RBD_SIDECAR_COUNT_4_12 = 14
```

Comment:
What is this? Can't recall anything with the name.

Reply:
In case of RDR operator deployment, there is a step where we need to check RBD sidecasr container count. In the versions <4.12 count was 18 but in 4.12 its 14. Hence this change.